pallets/nft/src/merkle_proof.rs

```diff
@@ -0,0 +1,21 @@
+use sp_core::hash::H256;
+use sp_core::hashing::keccak_256;
+
+/// Verify the given Merkle proof and Merkle root
+/// - Each pair of leaves and each pair of pre-images are assumed to be sorted.
+/// - With reference of https://docs.openzeppelin.com/contracts/4.x/api/utils#MerkleProof
+pub fn proof_verify(computed_hash: &[u8; 32], proof: &Vec<[u8; 32]>, root: &[u8]) -> bool {
```

Comment:
Should we use vec<u8> or [u8; 32] for CID

Reply:
fixed